extensions/interactions/ItemSelectionInput/directives/item-selection-input-rules.service.ts

```diff
@@ -28,50 +26,31 @@ import { ItemSelectionRuleInputs } from 'interactions/rule-input-defs';
   providedIn: 'root'
 })
 export class ItemSelectionInputRulesService {
-  private removeDuplicatesInArrayPipe: RemoveDuplicatesInArrayPipe = (
-    new RemoveDuplicatesInArrayPipe());
-
   Equals(
       answer: ItemSelectionAnswer,
       inputs: ItemSelectionRuleInputs): boolean {
-    var normalizedAnswer = this.removeDuplicatesInArrayPipe.transform(answer);
-    var normalizedInput = this.removeDuplicatesInArrayPipe.transform(inputs.x);
-    return normalizedAnswer.length === normalizedInput.length &&
-        normalizedAnswer.every((val) => {
-          return normalizedInput.indexOf(val) !== -1;
-        });
+    return answer.length === inputs.x.length &&
```

Comment:
I don't quite follow -- why is it OK to remove the "remove the duplicates" step for all these rules?

Reply:
I think it's just belt-and-suspenders -- you do have validation checks but these only get applied on-publish right? So what happens if you preview and play an exploration (via the exploration preview tab) while it's still in an invalid state?